lib/membrane_h264_plugin/parser/nalu_splitter.ex

```diff
@@ -6,7 +6,16 @@ defmodule Membrane.H264.Parser.NALuSplitter do
   The splitting is based on
   "Annex B" of the "ITU-T Rec. H.264 (01/2012)".
   """
-  alias Membrane.H264.Parser.NALu
+
+  @type t :: %__MODULE__{unparsed_payload: binary()}
```

Comment:
If NALuParser.t() is opaque I would follow it and make NALuSplitter and AUSplitter.t() opaques too

Reply:
`AUSplitter.t` is already opaque, but I have just made `NALuSplitter.t` opaque